test/unit/org/apache/cassandra/io/sstable/ULIDBasedSSTableIdGeneratorTest.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
nit: checklist says new files should have the DS copyright header

Reply:
won't be my last time this catches me…

Reply:
Same!